doc/source/ray-core/configure.rst

```diff
@@ -149,7 +149,6 @@ In addition to ports specified above, the head node needs to open several more p
 - ``--port``: Port of Ray (GCS server). The head node will start a GCS server listening on this port. Default: 6379.
 - ``--ray-client-server-port``: Listening port for Ray Client Server. Default: 10001.
 - ``--redis-shard-ports``: Comma-separated list of ports for non-primary Redis shards. Default: Random values.
```

Comment:
so this change is not backward compatible?

Reply:
We still accept this arg, just no longer use it, so existing workload won't crash (will receive a warning)